examples/material_gallery/lib/demo/selection_controls_demo.dart

```diff
@@ -4,74 +4,241 @@
+                  checkboxValueB = value;
+                });
+              })
+            ]
           ),
           new Row(
-            children: <int>[0, 1, 2].map((int i) {
-              return new Radio<int>(
-                value: i,
-                groupValue: _radioValue,
-                onChanged: _setRadioValue
-              );
-            }).toList(),
-            mainAxisAlignment: MainAxisAlignment.spaceAround
-          ),
+            mainAxisAlignment: MainAxisAlignment.collapse,
+            children: <Widget>[
+              new Checkbox(value: true),
+              new Checkbox(value: false)
+            ]
+          )
+        ]
+      )
+    );
+  }
+
+  Widget buildRadio() {
+    return new Align(
+      alignment: new FractionalOffset(0.5, 0.4),
+      child: new Column(
+        mainAxisAlignment: MainAxisAlignment.collapse,
+        children: <Widget>[
           new Row(
-            children: <int>[0, 1].map((int i) {
-              return new Radio<int>(value: i, groupValue: 0); // Disabled
-            }).toList(),
-            mainAxisAlignment: MainAxisAlignment.spaceAround
+            mainAxisAlignment: MainAxisAlignment.collapse,
+            children: <Widget>[
+              new Radio<int>(
+                value: 0,
+                groupValue: radioValue,
+                onChanged: _setRadioValue
+              ),
+              new Radio<int>(
+                value: 1,
+                groupValue: radioValue,
+                onChanged: _setRadioValue
+              ),
+              new Radio<int>(
+                value: 2,
+                groupValue: radioValue,
+                onChanged: _setRadioValue
+              )
+            ]
           ),
           new Row(
+            mainAxisAlignment: MainAxisAlignment.collapse,
             children: <Widget>[
-              new Switch(value: _switchValue, onChanged: _setSwitchValue),
-              new Switch(value: false), // Disabled
-              new Switch(value: true), // Disabled
-            ],
-            mainAxisAlignment: MainAxisAlignment.spaceAround
-          ),
-        ],
-        mainAxisAlignment: MainAxisAlignment.spaceAround
+              new Radio<int>(
+                value: 0,
+                groupValue: 0
+              ),
+              new Radio<int>(
+                value: 1,
+                groupValue: 0
+              ),
+              new Radio<int>(
+                value: 2,
+                groupValue: 0
+              )
+            ]
+          )
+        ]
+      )
+    );
+  }
+
+  Widget buildSwitch() {
+    return new Align(
+      alignment: new FractionalOffset(0.5, 0.4),
+      child: new Row(
+        mainAxisAlignment: MainAxisAlignment.collapse,
+        children: <Widget>[
+          new Switch(value: switchValue, onChanged: (bool value) {
+            setState(() {
+              switchValue = value;
+            });
+          }),
+          new Switch(value: true),
```

Comment:
The original code, here and elsewhere, included a `// Disabled` comment for these cases. Seems reasonable to keep that.